src/elements/x-safe.css

```diff
@@ -341,20 +336,23 @@ nav.actions.floating-actions {
     opacity: 0.6;
 }
 
-.floating-btn [new-tx] {
-    background-image: url('data:image/svg+xml,<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path fill="%234463D7" d="M15,20H9V12H4.16L12,4.16L19.84,12H15V20Z" /></svg>');
+.floating-btn button {
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
 }
 
-.floating-btn [new-tx]:disabled {
-    background-image: url('data:image/svg+xml,<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path fill="%234463D7" opacity="0.6" d="M15,20H9V12H4.16L12,4.16L19.84,12H15V20Z" /></svg>');
+.floating-btn button > div {
+    display: flex;
```

Comment:
I suggest keeping the icons as `background-image`, then they can be centered by `background-position` without the need for the additional div.

Reply:
see above

Reply:
Changed it back to `background-image`